### DISCUSSION TITLE
2.4 Beta Playbook rules

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#10081] [2.4] [answered]
Will the playbook rules work with the current Elastic Agent log setup or will it need to be adjusted?

**Comment by defensivedepth** [comment, marked as answer]
We are in the process of adjusting them, they will not work consistently during Beta 1.

**Reply by ColeVan** [reply]
Thanks for the reply! Looking forward to more greatness out of 2.4!
